alws/models.py

```diff
@@ -911,6 +911,10 @@ class TestTask(Base):
         nullable=True
     )
     repository = relationship('Repository')
+    scheduled_at = sqlalchemy.Column(sqlalchemy.DateTime, nullable=True)
+    started_at = sqlalchemy.Column(sqlalchemy.DateTime, nullable=True)
+    finished_at = sqlalchemy.Column(sqlalchemy.DateTime, nullable=True)
+    stats = sqlalchemy.Column(JSONB, nullable=True)
```

Comment:
why we create a stats field instead of adding new foreign key in `PerformanceStats`?